test/outreach/connector.go

```diff
@@ -0,0 +1,56 @@
+package outreach_test
```

Comment:
this package should be called `outreach` to be consistent with the other packages in this `test` directory.